### PR TITLE
Feature http headers

### PR DESCRIPTION
**Pull request by ddemoss222**
Implement feature flag to pass headers to Puppeteer via "headers". Also, add feature flag to disable headless mode via "headed".

### CODE REVIEW

File: decktape.js [hunk 1]
```diff
@@ -102,6 +102,16 @@ parser.script('decktape').options({
     transform : parseRange,
     help      : 'Range of slides to be exported, a combination of slide indexes and ranges (e.g. \'1-3,5,8\')',
   },
+  headed : {
+    default : false,
```
Comment (astefanutti) [inline decktape.js:106]: I'd suggest to stick to the current default value, which is `true`, and remove the negation when using the option below.

File: decktape.js [hunk 2]
```diff
@@ -102,6 +102,16 @@ parser.script('decktape').options({
     transform : parseRange,
     help      : 'Range of slides to be exported, a combination of slide indexes and ranges (e.g. \'1-3,5,8\')',
   },
+  headed : {
```
Comment (astefanutti) [inline decktape.js:105]: I'd suggest `headless`, WDYT?

**Review by astefanutti — CHANGES_REQUESTED** [review]
Thanks for your patience. I've left a couple of comments, then it'll be good to go.

File: decktape.js (hunk 2 above)
Comment (ddemoss222) [reply decktape.js]: I agree, that seems to better align with the puppeteer API

**Comment by ddemoss222** [comment]
Thank you for the comments. I have resubmitted PR

File: decktape.js [hunk 3]
```diff
@@ -102,6 +102,16 @@ parser.script('decktape').options({
     transform : parseRange,
     help      : 'Range of slides to be exported, a combination of slide indexes and ranges (e.g. \'1-3,5,8\')',
   },
+  headless : {
+    default : true,
```
Comment (astefanutti) [inline decktape.js:106]: Could you change it to the new default `new`, as Decktape has been updated to use https://developer.chrome.com/articles/new-headless/#new-headless-in-puppeteer.

**Review by astefanutti — APPROVED** [review]
Thanks!

**Pull request merged**